src/tactic/ui/startup/column_edit_wdg.py

```diff
@@ -103,7 +103,9 @@ def get_display(my):
                     names.push(name);
                 }
 
-                spt.table.add_columns(names)
+                // Unless there is a table here, we should not do this.
+                // Better handled with a callback
+                //spt.table.add_columns(names)
```

Comment:
There is no table usually.  It was added some time ago and it basically causes an alert every time.  And if there is, it is the last table used since this function does not switch the context to any table, so it adds columns to a completely unrelated table.